src/link.ts

```diff
@@ -0,0 +1,83 @@
+import { html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { createRef, ref } from 'lit/directives/ref.js';
+import packageJson from '../package.json' with { type: 'json' };
+import styles from './link.styles.js';
+import shadowRootMode from './library/shadow-root-mode.js';
+import final from './library/final.js';
+import required from './library/required.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'glide-core-link': GlideCoreLink;
+  }
+}
+
+/**
+ * @attr {string} label
+ * @attr {string} url
+ * @attr {boolean} [disabled=false]
+ * @attr {string} [download]
+ * @attr {'_blank'|'_parent'|'_self'|'_top'} [target]
```

Comment:
Just a basic set of attributes so far. `target` strikes me as an obvious. `download` less so but still probably a given. What do you think?